### DISCUSSION TITLE
Terminate a poll with a selected date

### DISCUSSION THREAD

**Discussion by mwithi** [lukevella/rallly#376] [General]
Hello all, great alternative to other softwares/platforms!

How to close a poll? I cannot find a way or a button to do that, like in this example:

![image](https://user-images.githubusercontent.com/2938553/208082135-c78e7ca2-b693-4a4f-9cf8-869b2f0fe167.png)

**Comment by lukevella** [comment]
Hi @mwithi, totally agree that this would be super useful and it's definitely something you can expect to see. It will come after the next major release though which has a target release date of Jan/Feb 2023  🤞

**Reply by mwithi** [reply]
Thanks! I will watch the project! Cheers!
